winter-website.css

```diff
@@ -1,4 +1,70 @@
 body {
+	padding: 0px;
+	margin: 0px;
 	color: #e8f3fc;
 	background-color: black;
+	font-family: sans-serif;
+}
+
+*, *::before, *::after {
+   box-sizing: border-box;
+}
+
+.navBar {
+	display: flex;
+	justify-content: space-between;
+	width: auto;
+	padding: 20px;
+	margin: 20px;
+	background-color: #5d87cf;
+	opacity: 0.5;
+}
+
+.navLink {
+	display: flex;
+	color: #e8f3fc;
+	text-decoration: none;
+}
+
+h1 {
+	font-size: 150px;
+	padding: 0;
+	margin: 0;
+}
+
+.sectionAll {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	position: absolute;
```

Comment:
risky using absolute position, but without seeing full page not sure what it is doing when youi resize etc. Maybe check it out in inspect on browser and see if it breaks if you tick it off

Reply:
removed these - you're right -- they didn't need these positions!